content/en/docs/pipelines/installation/overview.md

```diff
@@ -169,6 +180,13 @@ Upgrade Support (**Alpha**)
 
 For upgrade via reinstallation (with existing data) instructions, refer to [Upgrading AI Platform Pipelines](/docs/gke/pipelines/upgrade/#ai-platform-pipelines).
 
+Google Cloud Integrations
+:
+  * You can deploy AI Platform Pipelines on [Google Cloud Console UI](https://console.cloud.google.com/marketplace/details/google-cloud-ai-platform/kubeflow-pipelines).
+  * A Kubeflow Pipelines public endpoint with auth support is **auto-configured** for you.
+  * (Optional) You can choose to persist your data in Google Cloud managed storages (Cloud SQL and Cloud Storage).
```

Comment:
Minor proposal (as before):
- Hyphenate

```diff
- * ... your data in Google Cloud managed storages
+ * ... your data in Google Cloud-managed storages
```

Reply:
Do not hyphenate Cloud-managed.

managed storages -> managed storage

or

managed storages -> manages storage services

Reply:
Right, adding "services" at the end sounds better